build.xml

```diff
@@ -170,7 +170,25 @@
   </exec>
  </target>
 
- <target name="phpunit" depends="prepare,composer" description="Run unit tests with PHPUnit">
+<target name="start-selenium-server"  description="selenium-server-starts">
+  
+   <exec executable="xvfb-run" spawn="true">
+    <arg value="java"/>
+    <arg value="-jar"/>
+    <arg value="${basedir}/build/selenium-server-standalone-2.45.0.jar"/>
```

Comment:
if we are going to use selenium from build folder then we have to commit it. its better to commit it. what is its size?